staging/src/k8s.io/client-go/tools/clientcmd/loader.go

```diff
@@ -283,20 +281,14 @@ func (rules *ClientConfigLoadingRules) Migrate() error {
 			return fmt.Errorf("cannot migrate %v to %v because it is a directory", source, destination)
 		}
 
-		in, err := os.Open(source)
+		data, err := ioutil.ReadFile(source)
 		if err != nil {
 			return err
 		}
-		defer in.Close()
-		out, err := os.Create(destination)
+		err = ioutil.WriteFile(destination, data, 0666)
```

Comment:
why these permissions?

Reply:
This is what `os.Create()` uses under the hood https://github.com/golang/go/blob/go1.15.5/src/os/file.go#L316

Reply:
> it is created with mode 0666 (**before umask**)

Can you add a comment that this is umask'ed? I know our security ppl. finding this and complain.

Reply:
Done + rebased.